rtlsdr_wsprd.c

```diff
@@ -794,10 +805,20 @@ int main(int argc, char **argv) {
     uint32_t nLoop = 0;
 
     if (argc <= 1)
-        usage();
+        usage(stderr, 1);
 
-    while ((opt = getopt(argc, argv, "f:c:l:g:ao:p:u:d:n:i:tw:r:HQS")) != -1) {
+    while ((opt = getopt_long(argc, argv, short_options, long_options, &option_index)) != -1) {
```

Comment:
`getopt_long` is definitely a nice thing to add!
Good point, and adding real long option/names could be nice too.